web/src/search/input/SearchPage.tsx

```diff
@@ -87,6 +87,7 @@ export const SearchPage: React.FunctionComponent<Props> = props => {
     return (
         <div className="search-page">
             <BrandLogo className="search-page__logo" isLightTheme={props.isLightTheme} />
+            {props.isSourcegraphDotCom ? <div className="search-page__cloud-tag-line">Search public code</div> : null}
```

Comment:
```suggestion
            {props.isSourcegraphDotCom && <div className="search-page__cloud-tag-line">Search public code</div>}
```

is slightly easier to read I think :)

Reply:
Should be fixed.